packages/mdl-theme/README.md

```diff
@@ -0,0 +1,106 @@
+# mdl-theme
```

Comment:
Added a whole bunch of documentation covering Sass and custom properties, as well as a deeper explanation of what the module does. How does this look?